tests/system/conftest.py

```diff
@@ -40,18 +39,18 @@ def provider_env_vars():
 
 
 @pytest.fixture(autouse=True)
-def skip_if_env_var_not_set(provider_env_vars):
+def skip_if_env_var_not_set(provider_env_vars: list[str]) -> None:
     for env in itertools.chain(REQUIRED_ENV_VARS, provider_env_vars):
         if env not in os.environ:
             pytest.skip(f"Missing required environment variable {env}")
             return
 
 
-def pytest_collection_modifyitems(config, items):
+def pytest_collection_modifyitems(config: pytest.Config, items: list[pytest.Item]) -> None:
     """Add @pytest.mark.system(provider_name) for every system test."""
-    rootdir = Path(config.rootpath)
+    rootdir = config.rootpath
```

Comment:
👍 `rootpath` is already a Path: https://docs.pytest.org/en/6.2.x/reference.html#pytest.config.Config.rootpath